LayoutTests/inspector/css/getComputedPrimaryFontForNode-expected.txt

```diff
@@ -61,3 +66,28 @@ PASS: Width axis minimum should be '0'.
 PASS: Width axis maximum should be '1000'.
 PASS: Width axis default should be '750'.
 
+-- Running test setup.
+PASS: Font should be loaded.
+-- Running test case: CSS.getComputedPrimaryFontForNode.SyntheticWeight1
+PASS: Font should not have a synthesized weight.
```

Comment:
NIT: It's a bit odd that a test with the name `SyntheticWeight1` doesn't have a synthetic weight.  Perhaps (re)name this something like `CSS.getComputedPrimaryFontForNode.HasWeight` instead?

ditto below